owca/config.py

```diff
@@ -165,14 +250,33 @@ def register(cls):
         foo = Foo.__new__(Foo)  # construct uninitialized (blank) instance of given class
         foo.__init__(x=1) # initialize instance
 
+
+    Can be used both as function, decorator or parametrized decorator:
```

Comment:
```suggestion
    Can be used as function or decorator.
```
As far as I understand you can pass arguments to any decorator that accepts arguments.